dtdl-tools/src/lib.rs

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+// SPDX-License-Identifier: MIT
+
+// This lib.rs is for running the .NET unit tests in this digital_twins_connector folder.
+// Running 'cargo test' will run all the .NET unit tests and the Rust unit tests.
+#[cfg(test)]
+mod digital_twins_connector_dotnet_tests {
+    use std::io::{self, Write};
+    use std::path::Path;
+    use std::process::Command;
+
+    // The manifest directory is the directory that contains the Cargo.toml file for this crate.
+    const MANIFEST_DIR: &str = env!("CARGO_MANIFEST_DIR");
+
+    const DTDL_VALIDATOR_FILENAME: &str = "dtdl-validator";
+    const DTDL_VALIDATOR_BIN_DIR: &str = "target/debug/dtdl-validator/bin/Debug/net7.0";
```

Comment:
this will not be correct for non-debug builds. you'll need to either figure out which environment variable(s) you need from [here](https://doc.rust-lang.org/cargo/reference/environment-variables.html) or only enable this module when `debug_assertions` is enabled as described [here](https://stackoverflow.com/questions/39204908/how-to-check-release-debug-builds-using-cfg-in-rust)

Reply:
I have fixed this to use the rust OUT_DIR.